core/src/main/java/dev/keva/core/aof/AOFContainer.java

```diff
@@ -18,20 +18,21 @@
 @Component
 public class AOFContainer {
     private ReentrantLock bufferLock;
-    private List<Command> buffer;
     private ObjectOutputStream output;
+    private FileDescriptor fd;
+    private boolean alwaysFlush;
 
     @Autowired
     private KevaConfig kevaConfig;
 
     public void init() {
+        alwaysFlush = kevaConfig.getAofInterval() == 0;
```

Comment:
@the123saurav Can you replace all the `kevaConfig.getAofInterval() != 0` to `alwaysFlush `

Reply:
Line 49:

```java
if (kevaConfig.getAofInterval() != 0) {
            ScheduledExecutorService executorService = Executors.newSingleThreadScheduledExecutor();
```

Reply:
Sure